src/NUnitFramework/framework/Attributes/TestCaseSourceAttribute.cs

```diff
@@ -33,8 +33,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// TestCaseSourceAttribute indicates the source to be used to
-    /// provide test cases for a test method.
+    /// Identifies the source to be used as the test cases for a test method.
```

Comment:
Not sure about the re-wording here - the source and the test cases are different things, really. Can we switch this one back to the old version? `Identifies the source to be used to provide test cases for a test method.`